lib/challenge.js

```diff
@@ -61,6 +88,17 @@ var some_pet;
 var Pet;
 // Answer Starts Here
 
+var Pet = function(name, species, breed, noise) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise;
+  this.makeNoise = function(){
+    return "This pet makes a " + this.noise + ", noise.";
+  };
+}
```

Comment:
This is a good constructor function, and the `makeNoise` method is correct. However, this question asks you to create a prototype method. A little over half credit.